packages/devp2p/src/dpt/dpt.ts

```diff
@@ -1,4 +1,4 @@
-import ms from 'ms'
+import ms = require('ms')
```

Comment:
does this work?

Reply:
It does.

Neither import `ms from 'ms'` or `import * as ms from 'ms'` would work.  I don't know what is different about 'ms' but from experimenting this is what worked.